tests/cnf/core/network/internal/netenv/sriov.go

```diff
@@ -15,13 +15,14 @@ import (
 func WaitForSriovAndMCPStable(
 	apiClient *clients.Settings, waitingTime, stableDuration time.Duration, mcpName, sriovOperatorNamespace string) error {
 	glog.V(90).Infof("Waiting for SR-IOV and MCP become stable.")
+	time.Sleep(10 * time.Second)
```

Comment:
Do we want to add a comment on why we are waiting 10 seconds?

Reply:
This sleep is not really related to the newly created feature; it's already in 4.15 and 4.16. I've just added it to be consistent across all versions.

Reply:
Comment has been added